app/controllers/tweeets_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TweeetsController < ApplicationController
+  before_action :set_tweeet, only: %i[show edit update destroy]
+  before_action :authenticate_user!, except: %i[index show]
+
+  # GET /tweeets or /tweeets.json
+  def index
+    @tweeets = Tweeet.all.order('created_at DESC')
+    @tweeet = Tweeet.new
+  end
+
+  # GET /tweeets/1 or /tweeets/1.json
+  def show; end
+
+  # GET /tweeets/new
+  def new
+    @tweeet = current_user.tweeets.build

```

Comment:
It should initialize an empty post `@post = Post.new` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_new_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))